app/uk/gov/hmrc/customs/declarations/metrics/model/models.scala

```diff
@@ -16,11 +16,41 @@
 
 package uk.gov.hmrc.customs.declarations.metrics.model
 
-import java.time.ZonedDateTime
+import java.time.{Instant, ZoneId, ZonedDateTime}
 import java.util.UUID
 
-import play.api.libs.json._
 import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+object RequestReads {
+  val dateTimeRequestReads: Reads[ZonedDateTime] = JsPath.read[String].map { zonedDateTime =>
+    ZonedDateTime.parse(zonedDateTime)
+  }
+
+  val eventRequestReads: Reads[Event] = (
+    (__ \ "eventType").read[EventType] and
+    (__ \ "eventStart").read[ZonedDateTime](dateTimeRequestReads) and
+    (__ \ "eventEnd").read[ZonedDateTime](dateTimeRequestReads)) (Event.apply _)
+
+  val conversationMetricRequestReads: Reads[ConversationMetric] = (
+    (__ \ "conversationId").read[ConversationId] and eventRequestReads
+    )(ConversationMetric.apply _)
+}
+
```

Comment:
I think some kind of comment is needed here that talks about serialisation of `BSONDateTime` . Maybe also a [link](http://reactivemongo.org/releases/0.1x/documentation/json/overview.html#documents-and-values) that would explain why `$date` is used

Reply:
Added comment